### ISSUE TITLE
Updated HA 2021.4.1-4 502 Bad Gateway nginx

### ISSUE THREAD

**Issue by Goddur** [hassio-addons/addon-node-red#972]
# Problem/Motivation
Updated series of upgrades ending in core-2021.4.3
> (Why the issue was filed)
After updating HA Node-Red reported an error.
## Expected behavior
Node-Red should show up and display Nodes and Flows
> (What you expected to happen)
Expect NR to start but it does not only shows the 502 Bad Gateway error.

> (What actually happened)
Attempts at starting NR end up in a screen with 502 Bad Gateway nginx

Here is what the error in the Node-Red log file reports
[11:20:15] INFO: Starting Node-RED...
> addon-node-red@ start /opt
> node $NODE_OPTIONS node_modules/node-red/red.js "--settings" "/etc/node-red/config.js"
Error loading settings file: /etc/node-red/config.js
/config/node-red/settings.js:175
/ Configure the logging output
^
SyntaxError: Unexpected token '/'
    at wrapSafe (internal/modules/cjs/loader.js:979:16)
    at Module._compile (internal/modules/cjs/loader.js:1027:27)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1092:10)
    at Module.load (internal/modules/cjs/loader.js:928:32)
    at Function.Module._load (internal/modules/cjs/loader.js:769:14)
    at Module.require (internal/modules/cjs/loader.js:952:19)
    at require (internal/modules/cjs/helpers.js:88:18)
    at Object.<anonymous> (/etc/node-red/config.js:1:16)
    at Module._compile (internal/modules/cjs/loader.js:1063:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1092:10)

Here is the Node-Red Configuration file.
credential_secret: removed
dark_mode: true
http_node:
  username: myname
  password: removed
http_static:
  username: myname
  password: removed
ssl: false
certfile: fullchain.pem
keyfile: privkey.pem
require_ssl: false
system_packages: []
npm_packages: []
init_commands: []

**Comment by sinclairpaul** [comment]
> /config/node-red/settings.js:175
/ Configure the logging output
^

Looks like there is an issue in that file (the config folder is your HA config folder), has it been manually changed at all, what is at that line number as it should be a double / i.e.:

```
  // Configure the logging output
```

**Comment by Goddur** [comment]
> > /config/node-red/settings.js:175
> > / Configure the logging output
> > ^
> 
> Looks like there is an issue in that file (the config folder is your HA config folder), has it been manually changed at all, what is at that line number as it should be a double / i.e.:
> 
> ```
>   // Configure the logging output
> ```

Yes, I saw that, and knowing I have not modified files, folders, or paths not sure why that is reporting that as it does.

**Comment by sinclairpaul** [comment]
Well can you see what that line is configured as?

**Comment by Goddur** [comment]
> Well can you see what that line is configured as?

After looking at that line again I realized (with your help) that 175 was a reference to the line number in the settings.js file. Once at line 175 I could see this line / Configure the logging output I changed it to //Configure the logging output. Now Node-Red comes up. For those following, I did nothing to this file in fact learned of its existence through this failure. Thank you sinclairpaul for your help.
